benchmarks/microbenchmarks/operator_inp_logs/hf_train/AlbertForMaskedLM_training.txt

```diff
@@ -0,0 +1,115 @@
+Operator: aten._log_softmax.default
+cnt: 1, '((T([2048, 30000], th.float16), 1, False), {})'
```

Comment:
Since I'm not going to get a chance to bikeshed this later, I'm going to bikeshed this some more. How about...

```
F(T(2048, 30000, f16), 1, False)  # count: 1
```

Main changes:
* Use actual function call syntax, so you don't have to waste characters on arg tuple and kwarg dict. If you like, we can do it in a wordy way aka aten._log_softmax.default(T(2048, 30000, f16), 1, False). One benefit for the wordy format is you only one test per line, rather than having to deal with the hierarchical structure you currently have
* Support omitting the list/tuple in the tensor constructor, like our real constructors, saves two characters
* Use the shortened dtype names in https://github.com/pytorch/pytorch/blob/71141c30232d436ad61d2931af8808e4451f8cde/test/test_meta.py#L51-L65 They're used in Rust and I'm fond of the shortening
```

Reply:
Used shortened dtype names..